src/backend/distributed/metadata/distobject.c

```diff
@@ -392,7 +420,7 @@ UnmarkObjectDistributed(const ObjectAddress *address)
  * IsObjectDistributed returns if the object addressed is already distributed in the
  * cluster. This performs a local indexed lookup in pg_dist_object.
  */
-static bool
+bool
 IsObjectDistributed(const ObjectAddress *address)
```

Comment:
Instead of externing this, let's use IsAnyObjectDistributed in the call-site by wrapping the input via list_make1 because this seems to be what've done so far.